runtime/requirements.txt

```diff
@@ -3,10 +3,12 @@
 
 holidays
 hyperopt
+koalas
```

Comment:
Why do we need koalas?

Reply:
This will include spark as dependency. And for new spark, koalas is already in spark. We don't want to include this.

Reply:
I added it because I saw it's in the `environment.txt`. Is there a reason why we want it in `environemnt.txt` but not in `requriements.txt`?

Reply:
As per offline discussion, `environment.txt` is only used by github workflow for test set up. But we don't want to add koalas for the project dependencies with `requirements.txt`.